### DISCUSSION TITLE
Putty Fatal Error : Network error : Connection time out

### DISCUSSION THREAD

**Discussion by Chandanag24** [learndevopsonline/learndevopsonline#1664] [Q&A] [answered]
Hi, 
Previously i am facing an network error issue so that you were suggested to create an new instance  and i have created a new instance as workstationn and i try to connect it but same error i am facing...here below are my new instance details..
![Screenshot (7)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/be325267-c239-48ec-b963-66935f5a8ab7)
![1](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/f9797295-6a34-4b87-8bc4-6e32830bb137)
![2](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/0fc5a7df-c31b-485c-a50a-cbd389c141cf)
![3](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/85961040-0f23-4e6e-a2aa-d8cf1a53880a)

**Comment by sreedharm07** [comment]
@Chandanag24 
can you check once your instance is attached with allow all security group or not

**Reply by Chandanag24** [reply]
Yes, it is attached with allow-all
![4](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/014f27d4-947a-4013-b404-ae7786253f85)

**Comment by sreedharm07** [comment]
check internet gateway is attached to vpc or not 
if yes
check internet gateway was attached to the subnet or not in route tables

**Reply by Chandanag24** [reply]
I didn't create any vpc 's or anything i just created instance and i just want to connect this to putty.. but it is not connecting..

**Reply by sreedharm07** [reply]
To get internet to instance  internet gateway should attach to vpc and the subnets 
Actually by default internet gateway will be attached to subnets and vpc 

But in your case may be you deleted internet gateway by mistakenly, so you need to attach that manually to subnets and vpc
Please attach it it will works.

**Reply by Chandanag24** [reply]
yeah, just now i have attached vpc and subnets to internet gateway till i am not connecting it..

**Reply by sreedharm07** [reply]
ok
for this i want to check in your aws account 
can you give your aws account and password (snd to - sreedhar70755@gmail.com)
or 
if you dont want to share please join labsession, there directly can see into your aws account

**Reply by Chandanag24** [reply]
I shared my account mail & password to mentioned mail please check it.

**Reply by sreedharm07** [reply, marked as answer]
@Chandanag24 
internetgateway was not attached to route table 
i just attached it please try now it will works

**Reply by Chandanag24** [reply]
Yes. Now it is working fine.. I mistakenly attached another thing i think so it is not worked for me. Thank you @sreedharm07
